### PR TITLE
Polish Animations

### PR DESCRIPTION
**Pull request by jhrtn**
Working to address #32 

demo: https://macrodata-refinement-git-feature-updated-bin-animation-jhrtn.vercel.app/

### CODE REVIEW

**Comment by jhrtn** [comment]
Done some more tweaks to the easing and added the colours to the bin levels. The shader adds really interesting effect as the digits fade out

@shiffman Could you take a quick look and see if you're happy to merge this initial changes?

https://macrodata-refinement-git-feature-updated-bin-animation-jhrtn.vercel.app/

shader

no-shader

**Comment by shiffman** [comment]
Hey! This is fantastic! I'm getting some weird behavior, however, with the digits starting to move either before I release or perhaps with a long-ish delay after I release? So the interaction feels a little off. I'm happy to take a look at it to see if I can figure out why! Are you noticing this too? I can screen record an example if not.

**Comment by jhrtn** [comment]
I actually added that pause intentionally follow this reference:

https://user-images.githubusercontent.com/17256474/166161836-93d5e70f-c675-4838-9519-e43f1672830f.mp4

I added a variable `this.binPause` inside `data.js`. This is decremented inside `goBin()`, the scale of the digit increases and once it reaches zero we send the digits to the bin.

`binPause` is set to 30, so at 30fps that's a delay of 1 second.

The aim was to have the doors fully open before the numbers start moving. Happy to reduce this time if it feels off, or remove it altogether! 

Let me know and I can make the changes

File: data.js [hunk 1]
```diff
@@ -6,28 +6,35 @@ class Data {
     this.x = x;
     this.y = y;
     this.color = palette.FG; //TODO: pass this in as arg rather than global variable?
+    this.alpha = 255;
     this.sz = baseSize;
     this.refined = false;
     this.binIt = false;
     this.bin = undefined;
+    this.binPauseTime = 30;
```
Comment (jhrtn) [inline data.js]: This variable can be tweaked (or removed) to pause for a shorter period before sending digit to bin

**Comment by jhrtn** [comment]
I've reduced the pause time a bit from 30 frames to 8 which makes things feel a bit more snappy

 [macrodata-refinement-git-feature-updated-bin-animation-jhrtn.vercel.app](https://macrodata-refinement-git-feature-updated-bin-animation-jhrtn.vercel.app/)

https://user-images.githubusercontent.com/17256474/166239161-59a4200f-6ed0-43e0-b3bf-e91949971e6c.mp4

**Comment by shiffman** [comment]
Allright, let's try this we can always adjust as needed!

**Pull request merged**